Library/Formula/kafka.rb

```diff
@@ -27,10 +26,13 @@ def install
     inreplace "config/zookeeper.properties",
       "dataDir=/tmp/zookeeper", "dataDir=#{data}/zookeeper"
 
-    libexec.install %w(contrib core examples lib perf system_test)
+    # Workaround for conflicting slf4j-log4j12 jars (1.7.6 is preferred)
+    rm_f "core/build/dependant-libs-2.10.4/slf4j-log4j12-1.6.1.jar"
+
+    libexec.install %w(clients contrib core examples system_test)
 
     prefix.install "bin"
-    bin.env_script_all_files(libexec/"bin", :JAVA_HOME => "`/usr/libexec/java_home`")
+    bin.env_script_all_files(libexec/"bin", :JAVA_HOME => "${JAVA_HOME:=`/usr/libexec/java_home`}")
```

Comment:
Hmm, not sure what I think about this given the previous version is what we do for pretty much all other software.